docs/docs/stories.mdx

```diff
@@ -185,3 +187,64 @@ into a single file for evaluation. Read more about this format in [Testing Your
 This format is only used for testing and cannot be used for training.
 
 :::
+
+
+## End-to-end Training
+
+:::caution experimental feature
+End-to-end training is an experimental feature.
+We introduce experimental features to get feedback from our community, so we encourage you to try it out!
+However, the functionality might be changed or removed in the future.
+If you have feedback (positive or negative) please share it with us on the [Rasa Forum](https://forum.rasa.com).
+
+:::
+
+With end-to-end training, you do not have to deal with the specific
+intents of the messages that are extracted by the NLU pipeline
+or with separate `utter_` templates in the domain file.
+Instead, you can simply put the texts of the user and/or bot messages directly in the stories.
+See the [training data format](./training-data-format.mdx#end-to-end-training-stories)
+for detailed description of how to write end-to-end stories.
+
+Rasa end-to-end training is fully integrated with standard Rasa approach.
+It means that you can have mixed stories with some steps defined by actions
+and other steps defined by user or bot utterance directly.
+
+We call it end-to-end training because actual text is fed into policies.
+
+:::note nlu is ignored during e2e
+For end-to-end user inputs, intents classified by the NLU pipeline
+and extracted entities are ignored.
+
+:::
+
+Only [Rule Policy](./policies.mdx#rule-policy)
+and [TED Policy](./policies.mdx#ted-policy) allow end-to-end training.
+
+- `RulePolicy` uses simple string matching during prediction. Namely,
+  rules based on user text will only match if the user
+  text strings inside your rules and input during prediction are identical.
+
+- `TEDPolicy` passes user text through additional Neural Network to create
+  hidden representation of the text. In order to obtain robust performance you
+  need to create sufficient number of stories that have different user texts
+  during this dialogue turn.
+
+:::note next utterance selection
+Rasa policies are trained for next utterance selection.
+The only difference to creating `utter_` template is how `TEDPolicy` featurizes
+bot utterances.
+In case of a template, `TEDPolicy` sees only the name of the template, while
```

Comment:
```suggestion
In case of a template, `TEDPolicy` sees only the name of the template, while
```
```suggestion
In case of an `utter_` action, `TEDPolicy` sees only the name of the action, while
```